README.md

```diff
@@ -43,8 +43,7 @@ The library comes with a few tests like:
 
 ## Requirements
 
-GPUDirect RDMA requires NVIDIA Tesla or Quadro class GPUs based on Kepler,
-Pascal, Volta, or Turing, see [GPUDirect
+GPUDirect RDMA requires NVIDIA Data Center or Professional Graphics GPUs based on Kepler or newer generations, see [GPUDirect
```

Comment:
suggestion:
... Graphics (formely Tesla and Quadro) GPUs ...

Reply:
checked again and it seems we should use "NVIDIA RTX" to refer to professional graphics GPUs (https://www.nvidia.com/en-us/design-visualization/rtx/).
Let's add the links (so https://www.nvidia.com/en-us/data-center/ for data center) to disambiguate.